### ISSUE TITLE
Create a 1.5.2 hotfix release

### ISSUE THREAD

**Issue by oxarbitrage** [ZcashFoundation/zebra#8181]
We had problems with our last release (https://github.com/ZcashFoundation/zebra/pull/8165#issuecomment-1902616143)

- [ ] After we do https://github.com/ZcashFoundation/zebra/issues/8180 we need to create a 1.5.2 release and finish the release publishing process with that.
- [ ] Document removal of the feature in the hotfix changelog.

Blocked by https://github.com/ZcashFoundation/zebra/issues/8180

**Comment by oxarbitrage** [comment]
`cargo release publish --verbose --workspace --execute` was successful. All crates were published to crates.io.

**Comment by oxarbitrage** [comment]
Installation from crates.io work:

```
alfredo@spaceship:~/zebra/issue8180/main_with_1.5.2/zebra$ cargo install --locked  --force --version 1.5.2 zebrad && ~/.cargo/bin/zebrad
    Updating crates.io index
  Installing zebrad v1.5.2
    Updating crates.io index
    Updating crates.io index
   Compiling proc-macro2 v1.0.76
   Compiling unicode-ident v1.0.12
   Compiling libc v0.2.152
   Compiling cfg-if v1.0.0
   Compiling version_check v0.9.4
   Compiling autocfg v1.1.0
   Compiling serde v1.0.195
   Compiling once_cell v1.19.0
   Compiling subtle v2.4.1
   Compiling typenum v1.17.0
   Compiling log v0.4.20
   Compiling pin-project-lite v0.2.13
   Compiling memchr v2.7.1
   Compiling spin v0.5.2
   Compiling ppv-lite86 v0.2.17
   Compiling arrayvec v0.7.4
   Compiling tracing-core v0.1.32
   Compiling lazy_static v1.4.0
   Compiling byteorder v1.5.0
   Compiling generic-array v0.14.7
   Compiling radium v0.7.0
   Compiling crossbeam-utils v0.8.19
   Compiling tap v1.0.1
   Compiling nonempty v0.7.0
   Compiling wyz v0.5.1
   Compiling constant_time_eq v0.3.0
   Compiling futures-core v0.3.30
   Compiling arrayref v0.3.7
   Compiling futures-sink v0.3.30
   Compiling cpufeatures v0.2.12
   Compiling funty v2.0.0
   Compiling thiserror v1.0.56
   Compiling syn v1.0.109
   Compiling semver v1.0.21
   Compiling quote v1.0.35
   Compiling syn v2.0.48
   Compiling getrandom v0.2.12
   Compiling jobserver v0.1.27
   Compiling bitvec v1.0.1
   Compiling rand_core v0.6.4
   Compiling cc v1.0.83
   Compiling memuse v0.2.1
   Compiling rand_chacha v0.3.1
   Compiling num-traits v0.2.17
   Compiling ahash v0.8.7
   Compiling rand v0.8.5
   Compiling num_cpus v1.16.0
   Compiling blake2b_simd v1.0.2
   Compiling zerocopy v0.7.32
   Compiling either v1.9.0
   Compiling crypto-common v0.1.6
   Compiling block-buffer v0.10.4
   Compiling static_assertions v1.1.0
   Compiling digest v0.10.7
   Compiling crossbeam-epoch v0.9.18
   Compiling slab v0.4.9
   Compiling rayon-core v1.12.0
   Compiling bytes v1.5.0
   Compiling crossbeam-deque v0.8.5
   Compiling inout v0.1.3
   Compiling aho-corasick v1.1.2
   Compiling glob v0.3.1
   Compiling regex-syntax v0.8.2
   Compiling mio v0.8.10
   Compiling socket2 v0.5.5
   Compiling clang-sys v1.7.0
   Compiling signal-hook-registry v1.4.1
   Compiling futures-channel v0.3.30
   Compiling futures-io v0.3.30
   Compiling futures-task v0.3.30
   Compiling crunchy v0.2.2
   Compiling prettyplease v0.2.16
   Compiling ff v0.13.0
   Compiling group v0.13.0
   Compiling tinyvec_macros v0.1.1
   Compiling pin-utils v0.1.0
   Compiling tinyvec v1.6.0
   Compiling rayon v1.8.0
   Compiling allocator-api2 v0.2.16
   Compiling pairing v0.23.0
   Compiling itoa v1.0.10
   Compiling pasta_curves v0.5.1
   Compiling unicode-normalization v0.1.22
   Compiling hashbrown v0.14.3
   Compiling regex-automata v0.4.3
   Compiling num-integer v0.1.45
   Compiling minimal-lexical v0.2.1
   Compiling bls12_381 v0.8.0
   Compiling libloading v0.8.1
   Compiling nom v7.1.3
   Compiling secp256k1-sys v0.8.1
   Compiling sha2 v0.10.8
   Compiling universal-hash v0.5.1
   Compiling num-bigint v0.4.4
   Compiling ident_case v1.0.1
   Compiling opaque-debug v0.3.0
   Compiling equivalent v1.0.1
   Compiling smallvec v1.12.0
   Compiling libm v0.2.8
   Compiling maybe-rayon v0.1.1
   Compiling poly1305 v0.8.0
   Compiling regex v1.10.2
   Compiling cexpr v0.6.0
   Compiling serde_derive v1.0.195
   Compiling zeroize_derive v1.4.2
   Compiling tracing-attributes v0.1.27
   Compiling thiserror-impl v1.0.56
   Compiling tokio-macros v2.2.0
   Compiling futures-macro v0.3.30
   Compiling zeroize v1.7.0
   Compiling cipher v0.4.4
   Compiling jubjub v0.10.0
   Compiling chacha20 v0.9.1
   Compiling tokio v1.35.1
   Compiling futures-util v0.3.30
   Compiling tracing v0.1.40
   Compiling ring v0.16.20
   Compiling aead v0.5.2
   Compiling rustc-hash v1.1.0
   Compiling peeking_take_while v0.1.2
   Compiling shlex v1.2.0
   Compiling halo2_legacy_pdqsort v0.1.0
   Compiling bitflags v2.4.1
   Compiling base64ct v1.0.1
   Compiling lazycell v1.3.0
   Compiling password-hash v0.3.2
   Compiling halo2_proofs v0.3.0
   Compiling futures-executor v0.3.30
   Compiling chacha20poly1305 v0.10.1
   Compiling cbc v0.1.2
   Compiling incrementalmerkletree v0.5.0
   Compiling lock_api v0.4.11
   Compiling strsim v0.10.0
   Compiling platforms v3.3.0
   Compiling untrusted v0.7.1
   Compiling fnv v1.0.7
   Compiling serde_json v1.0.111
   Compiling darling_core v0.20.3
   Compiling fpe v0.6.1
   Compiling zcash_note_encryption v0.4.0
   Compiling futures v0.3.30
   Compiling pbkdf2 v0.10.1
   Compiling aes v0.8.3
   Compiling impl-trait-for-tuples v0.2.2
   Compiling hex v0.4.3
   Compiling uint v0.9.5
   Compiling indexmap v2.1.0
   Compiling reddsa v0.5.1
   Compiling rustc_version v0.4.0
   Compiling halo2_gadgets v0.3.0
   Compiling pin-project-internal v1.1.3
   Compiling curve25519-dalek v4.1.1
   Compiling bs58 v0.5.0
   Compiling hmac v0.12.1
   Compiling f4jumble v0.1.0
   Compiling blake2s_simd v1.0.2
   Compiling zcash_encoding v0.2.0
   Compiling ryu v1.0.16
   Compiling scopeguard v1.2.0
   Compiling byte-slice-cast v1.2.2
   Compiling bech32 v0.9.1
   Compiling zcash_address v0.3.1
   Compiling darling_macro v0.20.3
   Compiling pin-project v1.1.3
   Compiling bip0039 v0.10.1
   Compiling tokio-util v0.7.10
   Compiling curve25519-dalek-derive v0.1.1
   Compiling ripemd v0.1.3
   Compiling orchard v0.6.0
   Compiling equihash v0.2.0
   Compiling rustc-hex v2.1.0
   Compiling rustix v0.38.30
   Compiling pkg-config v0.3.28
   Compiling fixed-hash v0.7.0
   Compiling darling v0.20.3
   Compiling redjubjub v0.7.0
   Compiling link-cplusplus v1.0.9
   Compiling parity-scale-codec v3.6.9
   Compiling bindgen v0.65.1
   Compiling linux-raw-sys v0.4.12
   Compiling vcpkg v0.2.15
   Compiling getrandom v0.1.16
   Compiling tower-service v0.3.2
   Compiling signature v2.2.0
   Compiling ed25519 v2.2.3
   Compiling impl-codec v0.6.0
   Compiling primitive-types v0.11.1
   Compiling serde_with_macros v3.4.0
   Compiling bridgetree v0.4.0
   Compiling lz4-sys v1.9.4
   Compiling home v0.5.9
   Compiling portable-atomic v1.6.0
   Compiling libz-sys v1.1.14
   Compiling termcolor v1.4.1
   Compiling humantime v2.1.0
   Compiling bindgen v0.69.2
   Compiling unicode-width v0.1.11
   Compiling cxxbridge-flags v1.0.107
   Compiling iana-time-zone v0.1.59
   Compiling bitflags v1.3.2
   Compiling regex-syntax v0.6.29
   Compiling parking_lot_core v0.9.9
   Compiling which v4.4.2
   Compiling chrono v0.4.31
   Compiling cxx v1.0.107
   Compiling codespan-reporting v0.11.1
   Compiling serde_with v3.4.0
   Compiling zcash_history v0.3.0
   Compiling regex-automata v0.1.10
   Compiling x25519-dalek v2.0.0
   Compiling ed25519-zebra v4.0.3
   Compiling bitflags-serde-legacy v0.1.1
   Compiling serde-big-array v0.5.1
   Compiling displaydoc v0.2.4
   Compiling hdrhistogram v7.5.4
   Compiling itertools v0.12.0
   Compiling crossbeam-channel v0.5.11
   Compiling overload v0.1.1
   Compiling rustversion v1.0.14
   Compiling tower-layer v0.3.2
   Compiling console v0.15.8
   Compiling tower v0.4.13
   Compiling nu-ansi-term v0.46.0
   Compiling matchers v0.1.0
   Compiling rand_core v0.5.1
   Compiling cxx-gen v0.7.115
   Compiling libgit2-sys v0.16.1+1.7.1
   Compiling spin v0.9.8
   Compiling nanorand v0.7.0
   Compiling tracing-log v0.2.0
   Compiling sharded-slab v0.1.7
   Compiling thread_local v1.1.7
   Compiling percent-encoding v2.3.1
   Compiling number_prefix v0.4.0
   Compiling camino v1.1.6
   Compiling unicode-bidi v0.3.14
   Compiling option-ext v0.2.0
   Compiling dirs-sys v0.4.1
   Compiling idna v0.5.0
   Compiling tracing-subscriber v0.3.18
   Compiling indicatif v0.17.7
   Compiling form_urlencoded v1.2.1
   Compiling librocksdb-sys v0.11.0+8.1.1
   Compiling flume v0.10.14
   Compiling parking_lot v0.12.1
   Compiling rand_chacha v0.2.2
   Compiling bellman v0.14.0
   Compiling cxxbridge-macro v1.0.107
   Compiling metrics-macros v0.7.1
   Compiling unicase v2.7.0
   Compiling fastrand v2.0.1
   Compiling powerfmt v0.2.0
   Compiling known-folders v1.1.0
   Compiling httparse v1.8.0
   Compiling anyhow v1.0.79
   Compiling xdg v2.5.2
   Compiling utf8parse v0.2.1
   Compiling rlimit v0.10.1
   Compiling parking_lot_core v0.8.6
   Compiling http v0.2.11
   Compiling anstyle-parse v0.2.3
   Compiling metrics v0.21.1
   Compiling deranged v0.3.11
   Compiling tempfile v3.9.0
   Compiling vergen v8.3.0
   Compiling rand v0.7.3
   Compiling howudoin v0.1.2
   Compiling url v2.5.0
   Compiling dirs v5.0.1
   Compiling humantime-serde v1.1.1
   Compiling tokio-stream v0.1.14
   Compiling tracing-futures v0.2.5
   Compiling cargo-platform v0.1.6
   Compiling env_logger v0.7.1
   Compiling backtrace v0.3.69
   Compiling metrics v0.22.0
   Compiling zcash_script v0.1.14
   Compiling bstr v1.9.0
   Compiling instant v0.1.12
   Compiling eyre v0.6.11
   Compiling anstyle-query v1.0.2
   Compiling colorchoice v1.0.0
   Compiling adler v1.0.2
   Compiling try-lock v0.2.5
   Compiling num_threads v0.1.6
   Compiling time-core v0.1.2
   Compiling anstyle v1.0.4
   Compiling gimli v0.28.1
   Compiling globset v0.4.14
   Compiling anstream v0.6.7
   Compiling time v0.3.31
   Compiling want v0.3.1
   Compiling miniz_oxide v0.7.1
   Compiling quickcheck v0.9.2
   Compiling cargo_metadata v0.18.1
   Compiling http-body v0.4.6
   Compiling addr2line v0.21.0
   Compiling jsonrpc-core v18.0.0
   Compiling toml v0.5.11
   Compiling bincode v1.3.3
   Compiling tokio-util v0.6.10
   Compiling quickcheck_macros v0.9.1
   Compiling object v0.32.2
   Compiling fs-err v2.11.0
   Compiling unicode-xid v0.2.4
   Compiling hex-literal v0.4.1
   Compiling wagyu-zcash-parameters-3 v0.2.0
   Compiling indenter v0.3.3
   Compiling wagyu-zcash-parameters-1 v0.2.0
   Compiling wagyu-zcash-parameters-6 v0.2.0
   Compiling clap_lex v0.6.0
   Compiling heck v0.4.1
   Compiling wagyu-zcash-parameters-5 v0.2.0
   Compiling wagyu-zcash-parameters-2 v0.2.0
   Compiling cfg-if v0.1.10
   Compiling wagyu-zcash-parameters-4 v0.2.0
   Compiling rustc-demangle v0.1.23
   Compiling mset v0.1.1
   Compiling httpdate v1.0.3
   Compiling wagyu-zcash-parameters v0.2.0
   Compiling hyper v0.14.28
   Compiling net2 v0.2.39
   Compiling clap_derive v4.4.7
   Compiling clap_builder v4.4.18
   Compiling ordered-map v0.4.2
   Compiling synstructure v0.12.6
   Compiling proc-macro-crate v0.1.5
   Compiling jsonrpc-server-utils v18.0.0
   Compiling parking_lot v0.11.2
   Compiling tower-batch-control v0.2.41-beta.10
   Compiling tracing-error v0.2.0
   Compiling tower-fallback v0.2.41-beta.10
   Compiling toml_datetime v0.6.5
   Compiling serde_spanned v0.6.5
   Compiling owo-colors v3.5.0
   Compiling winnow v0.5.34
   Compiling jsonrpc-http-server v18.0.0
   Compiling color-eyre v0.6.2
   Compiling clap v4.4.18
   Compiling abscissa_derive v0.7.0
   Compiling toml_edit v0.21.0
   Compiling jsonrpc-derive v18.0.0
   Compiling secrecy v0.8.0
   Compiling wait-timeout v0.2.0
   Compiling tracing-log v0.1.4
   Compiling canonical-path v2.0.2
   Compiling arc-swap v1.6.0
   Compiling abscissa_core v0.7.0
   Compiling toml v0.8.8
   Compiling tracing-appender v0.2.3
   Compiling atty v0.2.14
   Compiling secp256k1 v0.26.0
   Compiling hdwallet v0.4.1
   Compiling zcash_primitives v0.13.0-rc.1
   Compiling zebra-chain v1.0.0-beta.34
   Compiling zcash_proofs v0.13.0-rc.1
   Compiling zebra-script v1.0.0-beta.34
   Compiling zebra-node-services v1.0.0-beta.34
   Compiling zebra-network v1.0.0-beta.34
   Compiling git2 v0.18.1
   Compiling zebrad v1.5.2
   Compiling rocksdb v0.21.0
   Compiling zebra-state v1.0.0-beta.34
   Compiling zebra-consensus v1.0.0-beta.34
   Compiling zebra-rpc v1.0.0-beta.34
    Finished release [optimized] target(s) in 4m 03s
   Replacing /home/alfredo/.cargo/bin/zebrad
    Replaced package `zebrad v1.5.2` with `zebrad v1.5.2` (executable `zebrad`)























































              X@8:::::::@X                      S888@t             S888%        
          @::X:;SX;%8S8@@:X::8              S:         .8      @.         .@    
       @:;@8@.S8  ;;t;. XX.;@@;:@         ;               %  ;               X  
     8:@88     .%@.S@.XS:     888:8      8                 t%                 @ 
    :;SX    XS8;::@X@8::;8t8:   8;;:@                                           
  @::@    88:::8 8::88::::::X@    8::8  @                                      8
 @:XS8   ;::::::8.88%8@S88::::8:  8S8:8 X                                      X
 ::;8 t8S8::::@St 88SX @SS@8:8S;8S @@:: 8                                      8
8:XX  %%@@::X% 8:  X@ t:8t X ..%t8  ;8:8                                        
::8t:  tX:X8tS 8: :@ ;.8@.X tt888:  @8:: X                                    % 
::8S:  @:8Xtt% 8: S 8 8S.   .8;::.  88::  %                                  t  
::X8   S@8   8:8.@8@8Xt S  .%8;S    %8::   8                                8   
 ::@8  S .   @:::::::X;@ 8.88S8    @8;:X     %                            %     
 8:X8X .@:X88:::::::::888 @;:X    %88::        %                        %       
  @::S    888@8:::::::::8X; 8     8::8           8                    8         
   X:;S@   tXX8;::::::::S::@8   8;S:8              S.              :t           
     ::8S@     :S88XX@8X:     @XX::                   %          %              
       8:;%X8.:S        t; X8%S:8X                      @      S                
         X8::8SS8SX8@@X@8;8S:8X                           S  %                  
              X@8:::::::@XS                                %                    

Thank you for running a mainnet zebrad 1.5.2 node!
You're helping to strengthen the network and contributing to a social good :)
2024-01-23T21:08:34.869501Z  INFO zebrad::components::tracing::component: started tracing component filter="info" TRACING_STATIC_MAX_LEVEL=LevelFilter::INFO LOG_STATIC_MAX_LEVEL=Info
2024-01-23T21:08:34.869594Z  INFO zebrad::components::tracing::component: set 'tracing.progress_bar ="summary"' in zebrad.toml to activate progress bars
2024-01-23T21:08:34.869612Z  INFO zebrad::application: Diagnostic Metadata:
version: 1.5.2
Zcash network: Mainnet
running state version: 25.3.0
initial disk state version: 25.3.0
features: default,getblocktemplate_rpcs,howudoin,indicatif,progress_bar,release_max_level_info
target triple: x86_64-unknown-linux-gnu
rust compiler: 1.74.0
rust release date: 2023-11-13
optimization level: 3
debug checks: false
2024-01-23T21:08:34.869625Z  INFO zebrad::application: loaded zebrad config config_path=None config=ZebradConfig { consensus: Config { checkpoint_sync: true }, metrics: Config { endpoint_addr: None }, network: Config { listen_addr: 0.0.0.0:8233, network: Mainnet, initial_mainnet_peers: {"dnsseed.z.cash:8233", "dnsseed.str4d.xyz:8233", "mainnet.seeder.zfnd.org:8233", "mainnet.is.yolo.money:8233"}, initial_testnet_peers: {"dnsseed.testnet.z.cash:18233", "testnet.seeder.zfnd.org:18233", "testnet.is.yolo.money:18233"}, cache_dir: IsEnabled(true), peerset_initial_target_size: 25, crawl_new_peer_interval: 61s, max_connections_per_ip: 1 }, state: Config { cache_dir: "/home/alfredo/.cache/zebra", ephemeral: false, delete_old_database: true, debug_stop_at_height: None, debug_validity_check_interval: None }, tracing: Config { inner: InnerConfig { use_color: true, force_use_color: false, filter: None, buffer_limit: 128000, endpoint_addr: None, flamegraph: None, progress_bar: None, log_file: None, use_journald: false } }, sync: Config { download_concurrency_limit: 50, checkpoint_verify_concurrency_limit: 1000, full_verify_concurrency_limit: 20, parallel_cpu_threads: 0 }, mempool: Config { tx_cost_limit: 80000000, eviction_memory_time: 3600s, debug_enable_at_height: None }, rpc: Config { listen_addr: None, parallel_cpu_threads: 0, debug_force_finished_sync: false }, mining: Config { miner_address: None, extra_coinbase_data: None, debug_like_zcashd: true } }
2024-01-23T21:08:34.869925Z  INFO {net="Main"}: zebrad::application: initialized rayon thread pool for CPU-bound tasks num_threads=12
2024-01-23T21:08:34.870850Z  INFO {net="Main"}: zebrad::commands::start: Starting zebrad
2024-01-23T21:08:34.871009Z  INFO {net="Main"}: zebrad::commands::start: initializing node state
2024-01-23T21:08:34.877108Z  INFO {net="Main"}: zebrad::commands::start: opening database, this may take a few minutes
2024-01-23T21:08:34.877285Z  INFO zebra_state::service::finalized_state::disk_format::upgrade: trying to open current database format running_version=25.3.0
2024-01-23T21:08:34.877525Z  INFO zebra_state::service::finalized_state::disk_db: the open file limit is high enough for Zebra current_limit=1024 min_limit=512 ideal_limit=1024
^C2024-01-23T21:08:39.477496Z  INFO {net="Main"}:sig{kind=SignalKind(2) name="SIGINT"}: zebrad::signal: received SIGINT, starting shutdown
2024-01-23T21:08:39.477607Z  INFO {net="Main"}: zebrad::components::tokio: waiting for async tokio tasks to shut down TOKIO_SHUTDOWN_TIMEOUT=20s
^C2024-01-23T21:08:40.531787Z  INFO zebra_state::service::finalized_state::disk_db: Opened Zebra state cache at /home/alfredo/.cache/zebra/state/v25/mainnet
2024-01-23T21:08:40.534959Z  INFO zebra_state::service::finalized_state::disk_format::upgrade: checking database format produced by a previous zebra instance is current and valid running_version=25.3.0
2024-01-23T21:08:40.534768Z  INFO zebra_state::service::finalized_state: loaded Zebra state cache tip=Some((Height(9751), block::Hash("000000009b28c88d493863549e760e833d6675f137e4927477f8f7b460a7e977")))
2024-01-23T21:08:40.536772Z  INFO zebra_state::service: starting legacy chain check
2024-01-23T21:08:40.537063Z  INFO zebra_state::service: cached state consensus branch is valid: no legacy chain found
2024-01-23T21:08:40.537173Z  INFO zebra_state::service::write: StateService closed the block reset channel. Is Zebra shutting down?
2024-01-23T21:08:40.537234Z  INFO zebra_state::service::finalized_state::disk_format::upgrade: checking new blocks were written in current database format running_version=25.3.0
2024-01-23T21:08:40.537243Z  INFO zebra_state::service::finalized_state::disk_format::upgrade: checking database format produced by new blocks in this instance is valid running_version=25.3.0
2024-01-23T21:08:40.552104Z  INFO zebra_state::service::finalized_state::disk_format::upgrade: database format is valid running_version=25.3.0 inital_disk_version=25.3.0
2024-01-23T21:08:40.552118Z  INFO zebra_state::service::finalized_state::disk_db: forcing shutdown of a state database with multiple active instances path="/home/alfredo/.cache/zebra/state/v25/mainnet"
2024-01-23T21:08:40.556447Z  INFO zebra_state::service: waiting for the block write task to finish
2024-01-23T21:08:40.556531Z  INFO zebra_state::service::finalized_state::disk_format::upgrade: checking new blocks were written in current database format running_version=25.3.0
2024-01-23T21:08:40.556538Z  INFO zebra_state::service::finalized_state::disk_format::upgrade: checking database format produced by new blocks in this instance is valid running_version=25.3.0
2024-01-23T21:08:40.568023Z  INFO zebra_state::service::finalized_state::disk_format::upgrade: database format is valid running_version=25.3.0 inital_disk_version=25.3.0
2024-01-23T21:08:40.570097Z  INFO {net="Main"}: zebrad::components::tokio: shutting down Zebra
2024-01-23T21:08:40.570111Z  INFO {net="Main"}: zebrad::commands::start: stopping zebrad
alfredo@spaceship:~/zebra/issue8180/main_with_1.5.2/zebra$ 
```
